app/scripts/modules/core/src/pipeline/executions/executionGroup/ExecutionGroup.tsx

```diff
@@ -66,7 +68,7 @@ export class ExecutionGroup extends React.Component<IExecutionGroupProps, IExecu
         CollapsibleSectionStateCache.isExpanded(sectionCacheKey),
       poll: null,
       canTriggerPipelineManually: !!pipelineConfig,
-      canConfigure: !!(pipelineConfig || strategyConfig),
+      canConfigure: !!(hasNonMPTV2PipelineConfig || strategyConfig),
```

Comment:
Will also lead to "Start Manual Execution" not being rendered for mptv2 pipelines. The UI does not currently support kicking off mptv2 pipeline executions so this works out. Going to follow up this PR with some work that addresses other parts of the application that incorrectly allow starting manual executions for mptv2 pipelines